scripts/validate_docstrings.py

```diff
@@ -527,8 +527,35 @@ def validate_one(func_name):
                                             err.message))
 
     if doc.is_function_or_method:
-        if not doc.returns and "return" in doc.method_source:
-            errs.append('No Returns section found')
+        if not doc.returns:
+            if "return" in doc.method_source:
+                errs.append('No Returns section found')
+        else:
+            returns_errs = []
+            if len(doc.returns) == 1 and doc.returns[0][1]:
+                returns_errs.append('The first line of the Returns section '
+                                    'should contain only the type, unless '
+                                    'multiple values are being returned.')
+            missing_desc, missing_cap, missing_period = False, False, False
+            for name, type_, desc in doc.returns:
+                desc = ''.join(desc)
+                missing_desc = missing_desc or not desc
+                missing_cap = missing_cap or desc and not desc[0].isupper()
+                missing_period = (missing_period
+                                  or desc and not desc.endswith('.'))
+            if missing_desc:
+                returns_errs.append('Return value has no description.')
+            if missing_cap:
+                returns_errs.append('Return value description should start '
+                                    'with a capital letter.')
+            if missing_period:
+                returns_errs.append('Return value description should finish '
+                                    'with ".".')
+            if returns_errs:
+                errs.append('Errors in Returns section')
```

Comment:
I assume you are emulating what you saw with the parameters section, but do we need this? Doesn't look like we are testing for it and I don't necessarily see the utility of it alongside the actual errors

Reply:
indeed, it's not as useful here as in the parameters section, as functions with multiple values are quite rare
I've removed this message and appended errors directly to `errs`